hyperspy/io.py

```diff
@@ -478,6 +478,9 @@ def save(filename, signal, overwrite=None, **kwds):
         ensure_directory(filename)
         if overwrite is None:
             overwrite = overwrite_method(filename)
+        if overwrite is False:
```

Comment:
This "fails" (as in does not overwrite) silently, and also might confuse future developers.

I would prefer this to be formatted along the lines:
```python
if overwrite is False and os.path.isfile(filename):
    raise ValueError("some appropriate text")
# If we get here, either we overwrite or the file does not exist
# Put the code that in your PR starts with line #485 
# (i.e. without the `if overwrite is True`)
```